src/styles/blocks/button.scss

```diff
@@ -0,0 +1,20 @@
+.button {
+  padding: 11px 0;
+
+  font-size: 14px;
+  font-weight: 700;
+  text-transform: uppercase;
+
+  color: #fff;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  cursor: pointer;
+
+  transition: all 0.2s linear;

```

Comment:
never add a transition for `all` properties. You need to add a transition properties that should change smoothly